vlm/src/main/scala/geotrellis/contrib/vlm/PaddedTile.scala

```diff
@@ -98,7 +98,7 @@ case class PaddedTile(chunk: Tile, colOffset: Int, rowOffset: Int, cols: Int, ro
 
   def foreachDouble(f: Double => Unit): Unit = {
     cfor(0)(_ < rows, _ + 1) { row =>
-      if (row < rowOffset || row > (rowOffset + rows - 1)) {
+      if (row < rowOffset || row > (rowOffset + chunk.rows) - 1) {
```

Comment:
Have you added a unit test? Or investigation.spec just started to work?

Reply:
Ah, `InvestigationSpec` passed, but then I removed it... I'll have to create a new unit test.